lib/go-tc/enum.go

```diff
@@ -100,6 +100,11 @@ func CacheTypeFromString(s string) CacheType {
 	return CacheTypeInvalid
 }
 
+const AllReadCapability = `all-read`
```

Comment:
I specifically fixed these to work, because the "API Tests" were breaking. That framework only assigns the "all" capabilities, and needs them to work to succeed.

Reply:
oh i see, so there are tests to check for the all-read and all-write capabilities? can you kill those tests because they are no longer relevant.

Reply:
Not quite. All the tests hit API endpoint using a role, which now needs capabilities (which also need "api capabilities"). All tests fail because they don't have permission and get Forbidden codes, without the code accepting the "all-read" and "all-write" capabilities given to the "API Test" user.

Reply:
@mitchell852 Changed, and made https://github.com/apache/trafficcontrol/issues/2793 to address the "API Tests" framework seeds.sql copy-paste.